docs/modules/user-guide/partials/proc_adding-schema-version-to-a-devfile.adoc

```diff
@@ -2,8 +2,13 @@
 = Adding schema version to a devfile
 
 [role="_abstract"]
-The `schemaVersion` attribute is mandatory in a devfile.
+The `schemaVersion` attribute is mandatory in a devfile. See the following list for `schemaVersion` properties in a devfile:
 
+[horizontal]
+Key:: schemaVersion
+Type:: string
+Required:: yes
+Description:: v2.1.0 of the schema
```

Comment:
This field should not tie to 2.1.0. Change the description to something like: "Version of the devfile schema used by the devfile"